lib/plsql/procedure.rb

```diff
@@ -104,7 +104,7 @@ def get_argument_metadata #:nodoc:
       @schema.select_all(
         "SELECT #{subprogram_id_column}, object_name, TO_NUMBER(overload), argument_name, position, data_level,
               data_type, in_out, data_length, data_precision, data_scale, char_used,
-              char_length, type_owner, type_name, type_subname
+              char_length, type_owner, type_name, type_subname, defaulted
```

Comment:
Yes, on 10g there's no such column, so I will do like in `subprogram_id_column`.